### PR TITLE
Fix clang type mismatch warnings in nvbios

### PR DESCRIPTION
**Pull request by Echelon9**
* nvbios: Fix printf() format specification in printscript()
* nvbios: Fix printf() format specification in main()

### CODE REVIEW

File: nvbios/nvbios.c [hunk 1]
```diff
@@ -758,7 +758,7 @@ void printscript (uint16_t soff) {
 				uint16_t iters = bios->data[soff+1];
 				uint32_t reg_off = soff + 3;
 				printcmd (soff, 3);
-				printf ("ZM_REG_SET_LOOP  0x%02hhx 0x%02hhx {\n", cnt, iters);
+				printf ("ZM_REG_SET_LOOP  0x%02hhx 0x%02hx {\n", cnt, iters);
```
Comment (imirkin) [inline nvbios/nvbios.c]: iters should be uint8_t. Note that it's only a single byte read from data.

File: nvbios/nvbios.c [hunk 2]
```diff
@@ -1691,7 +1691,7 @@ int main(int argc, char **argv) {
 
 			printf("-- ID = %u, ", i);
 			if (version == 0x20)
-				printf("mode: %i, link: %hhi, ", mode, link);
+				printf("mode: %i, link: %i, ", mode, link);
```
Comment (imirkin) [inline nvbios/nvbios.c]: same deal here. make link a uint8_t maybe.

File: nvbios/nvbios.c (hunk 1 above)
Comment (Echelon9) [inline nvbios/nvbios.c]: Will amend.

File: nvbios/nvbios.c (hunk 2 above)
Comment (Echelon9) [inline nvbios/nvbios.c]: Will amend.

**Pull request closed without merging**